quiche/src/recovery/mod.rs

```diff
@@ -251,6 +253,97 @@ impl Recovery {
     }
 }
 
+/// BBR settings used to customize the algorithm's behavior.
+///
+/// This is an experimental API.
+///
+/// A congestion control algorithm has dual-responsibility of effective network
+/// utilization and avoiding congestion. Custom values should be choosen
+/// carefully since incorrect values can lead to network degradation for all
+/// connections on the shared network.
+#[derive(Debug, Default, Copy, Clone, PartialEq)]
+#[repr(C)]
+pub struct BbrSettings {
```

Comment:
Would be good to be clear that this only works for gcongestion implementation.

I wonder if we should try to put this in gcongestion/mod.rs

Reply:
The public apis exposed in lib.rs mention that the overrides only apply to gcongestion. I think eventually we might want this to be generic across BBR impls which is why I placed it here.

Reply:
Let's put it in the gcongestion directory